package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "clearNextTemp": "rm -rf ./.next",
     "build": "npm run clearNextTemp && next build",
-    "dev": "npm run clearNextTemp && next dev -p 80",
+    "dev": "npm run clearNextTemp && next dev -p 8084",
```

Comment:
port 80 didn't work for me

Reply:
any clue why not? also, you can just change this in your local aliases (like `npm run dev -- -p 8084`), without modifying it for everyone else ;)

Reply:
I thought it was a typo. I'll update my aliases.

Reply:
The reason why I set it up this way is so that I can fake a domain (using the hosts file) so I could skip CORS issues etc.